exp/addons/api/v1alpha3/suite_test.go

```diff
@@ -29,29 +28,15 @@ import (
 )
 
 var (
-	env *envtest.Environment
+	env envtest.Environment
 	ctx = ctrl.SetupSignalHandler()
 )
 
 func TestMain(m *testing.M) {
-	// Bootstrapping test environment
 	utilruntime.Must(AddToScheme(scheme.Scheme))
```

Comment:
Can we remove this as well and just add it in the global envtest package?

Reply:
Yup, went through the usages. When we do it we always register the same scheme (and if we need it in the future we could make it parameterizable then)

Reply:
Hm took a closer look. Usually we just register the local scheme. I could implementing by adding the `AddToScheme` as an optional field of `RunInput`, but not sure if that's much better than just keeping it as is.

(I cannot just add the statement to the Run func because then I'll get a circular package dependency)